graphics/icon_gen.sh

```diff
@@ -43,10 +44,17 @@ file="$mydir/ic_launcher_sc.svg"
 export_files
 
 
+
 dpi=48 # 96/2
 file="$mydir/ic_launcher_foreground_sc.svg"
 export_files
 
 
+# As this is just for API > 33, we don't need to export to png mipmaps, we can just directly use the drawable
+monochrome_file_name="ic_launcher_monochrome_sc.xml"
+monochrome_input_file="$mydir/$monochrome_file_name"
+monochrome_output_dir="$mydir/../vector-app/src/main/res/drawable-anydpi-v26"
+cp "$monochrome_input_file" "$monochrome_output_dir/$monochrome_file_name"
```

Comment:
I tested ` ./alternative_package.sh "testing.fcm"` and it seems to pick this up and work as expected.